apps/cli/README.md

```diff
@@ -0,0 +1,21 @@
+# Cli
+
+**TODO: Add description**
```

Comment:
Can we remove this line/file? Or maybe put the CLI docs here? @andogro has been doing a great job helping us keep our documentation accurate and up to date. Let's remember to discuss our options next time we all meet.

Reply:
Yeah, I think getting some basic docs/info might be a good idea. I can write something simple.